config/config.example.js

```diff
@@ -1,6 +0,0 @@
-// Copy this file as config.js in the same folder, with the proper database connection URI.
```

Comment:
Thanks for the contribution! 🚀 Any reason you removed this file? Other than, it's a sample file for the real `config.js` file and that's in the `.gitignore`.